lib/Units.pm

```diff
@@ -77,13 +63,41 @@ our %known_units = (
 		'factor' => 1,
 		'degC'   => 1
 	},
+	"\x{00B0}C" => {    # Unicode degree symbol with text C
+		'factor' => 1,
+		'degC'   => 1
+	},
+	"\x{2103}" => {     # Unicode degree Celsius symbol
+		'factor' => 1,
+		'degC'   => 1
+	},
 	'degF' => {
 		'factor' => 1,
 		'degF'   => 1
 	},
+	"\x{00B0}F" => {    # Unicode degree symbol with text F
+		'factor' => 1,
+		'degF'   => 1
+	},
+	"\x{2109}" => {     # Unicode degree Fahrenheit symbol
+		'factor' => 1,
+		'degF'   => 1
+	},
+	'K' => {
+		'factor' => 1,
+		'K'      => 1
+	},
 	'degK' => {
 		'factor' => 1,
-		'degK'   => 1
+		'K'      => 1
+	},
+	"\x{00B0}K" => {    # Unicode degree symbol with text K
+		'factor' => 1,
+		'K'      => 1
+	},
```

Comment:
I strongly disagree with allowing students to use the incorrect unit.  It stores up problems for their futures.

Reply:
I don't disagree with you.